packages/portal/spec-renderer/src/components/SpecOperationsList.vue

```diff
@@ -130,6 +149,7 @@ import type { OperationListFilterFunction, Operation, OperationListItem, Tag } f
 import { v1 as uuidv1 } from 'uuid'
 import clonedeep from 'lodash.clonedeep'
 import composables from '../composables'
+import { DEFAULT_UNTAGGED_SECTION_NAME } from '../const'
```

Comment:
I don't think this one string really warrants a new file import, you could just define it statically `const var = 'default` here in the template

Related: we typically don't abbreviate the filenames here -- e.g. `constants.ts` 

but I don't think you need the separate file here since it's only used here

Reply:
I was following the pattern here: https://github.com/Kong/public-ui-components/blob/main/packages/core/copy-uuid/src/const.ts

I put it there because it IS a constant that we don't want people randomly changing without thought. I will rename to `constants.ts`.

Reply:
Ah ok makes sense 👍🏼 thanks for renaming 😄

Reply:
I also renamed the `const.ts` in `copy-uuid`